### ISSUE TITLE
Request to Add libcurl4 Package for cURL Support in Zoneminder Docker

### ISSUE THREAD

**Issue by joopdo** [zoneminder-containers/zoneminder-base#47]
I am an avid user of your Zoneminder Docker image and I have come across an issue regarding cURL support. The ESP32 Camera documentation specifies that in order to use the camera as a source in Zoneminder, the source type should be set to cURL (HTTP(S) only) and the URL should be configured as "http://ESP_IP_ADDRESS:8080/".

Unfortunately, the current version of the Zoneminder Docker image does not include support for cURL out of the box. However, I have discovered that after logging into the Docker image and manually installing the **_libcurl4_** package, cURL support is enabled and I am able to use the ESP32 Camera as a source successfully.

Therefore, I kindly request that you consider adding the libcurl4 package to the Zoneminder Docker image by default. This addition will greatly benefit users like me who rely on cURL for their camera sources.

Thank you for your attention to this matter, and I appreciate your efforts in maintaining the Zoneminder Docker repository.

**Comment by alexyao2015** [comment]
Anything related to dependencies should honestly be reported to zoneminder since it would either have to be a bug with zoneminder either not defining all the dependencies it needs or it being unable to find a dependency that it defined explicitly.

**Comment by alexyao2015** [comment]
Last nite here. I checked the build logs and I'm seeing libcurl4 as being installed so something else must be happening.

**Comment by joopdo** [comment]
I just double checked with a clean install. The error happens for me. 
![Screenshot from 2023-05-19 15-14-33](https://github.com/zoneminder-containers/zoneminder-base/assets/1182449/26268ae3-6f79-4c44-a192-9688fa3bde42)
Perhaps it's one of the dependencies that I install? 

apt install libcurl4 
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
The following additional packages will be installed:
  libnghttp2-14 librtmp1 libssh2-1
The following NEW packages will be installed:
  libcurl4 libnghttp2-14 librtmp1 libssh2-1

**Comment by alexyao2015** [comment]
Right. This is a bug with zoneminder's packaging in the latest stable release. It's not requesting it to be installed. If you use the main tag which builds from the latest master, it doesn't have this issue. It therefore should be reported to zoneminder to backport this fix, not here.

**Comment by joopdo** [comment]
Thank you @alexyao2015 ! I'll report this to zoneminder.
